eHealthAfrica/aether
# Aet 163 define api routing

#### What does this PR do?
adds `kongfig` as a component and defines apis for `kernel`, `odk`, `couch-sync`

#### How to test
`docker-compose -f ./docker-compose.yml up --build`
check that you can reach kernel from `http://localhost/aether/v1` or `kong.aether.local/aether/v1`

#### Related JIRA ticket
[AET-163](https://jira.ehealthafrica.org/browse/AET-163)
